src/assets/scripts/client/aircraft/AircraftStripView.js

```diff
@@ -386,6 +386,7 @@ export default class AircraftStripView {
         switch (navMode) {
             case WAYPOINT_NAV_MODE.FIX:
                 this.$heading.text(headingText);
+                this.$destination.text(destinationText.toUpperCase());
```

Comment:
sort of.  this is the (current) name of the aircraft strip element.  it is _sometimes_ the destination.  refer to the following screenshot for the various states and corresponding field names/purposes:
<img width="757" alt="screen shot 2017-04-22 at 1 21 32 pm" src="https://cloud.githubusercontent.com/assets/3325985/25319726/6170b3c6-2866-11e7-9761-210fdcd963d7.png">